Sources/Client/Model/Channel/Channel.swift

```diff
@@ -176,6 +184,10 @@ public final class Channel: Codable {
         cid = try container.decode(ChannelId.self, forKey: .cid)
         let members = try container.decodeIfPresent([Member].self, forKey: .members) ?? []
         self.members = Set<Member>(members)
+        
+        // TODO: properly?
+        self.memberCount = try container.decodeIfPresent(Int.self, forKey: .memberCount) ?? 0
+        
```

Comment:
Should we fail the decoding if member count is missing?

Reply:
It will failed decoding only if it will be presented as like this `member_count: "hmm"`. But I agree to set default value as `members.count` instead of `0`.